pkg/pyinst/bitmask.spec

```diff
@@ -33,6 +34,6 @@ if sys.platform.startswith("darwin"):
 		     name=os.path.join(
 		      'dist', 'Bitmask.app'),
                      appname='Bitmask',
-                     version='0.9.0rc2',
+                     version='0.9.2alpha1',
 		     icon='pkg/osx/bitmask.icns',
-		     bundle_identifier='bitmask-0.9.0rc2')
+		     bundle_identifier='bitmask-0.9.2alpha1')
```

Comment:
Maybe one day we can change this hardcoded versions form something generated from the makefile, getting the tag of the current commit and/or adding the commit-id or something there.